master/internal/command/command.go

```diff
@@ -356,7 +356,7 @@ func (c *command) Receive(ctx *actor.Context) error {
 	case job.SetGroupPriority:
 		err := c.setPriority(ctx, msg.Priority, true)
 		if err != nil {
-			ctx.Log().WithError(err)
+			ctx.Log().WithError(err).Info("setting command job priority")
```

Comment:
I still would just put the `ctx.Respond()` in the err block but definitely a nit pick lol. I have something against returning an `err` we know is nil from a readability perspective.